test/integration/features/user_creates_from_url.spec.js

```diff
@@ -0,0 +1,204 @@
+        createFromURLPage.visit(qs);
+        matchersHelpers.expectHeading(heading);
+      });
+
+      it('should load the image stream in to a newly created project', function(){
+        let createFromURLPage = new CreateFromURLPage();
+        createFromURLPage.visit(qs);
+        // NOTE: if the tabs are on the page because there are existing projects, click the 'Create New Project' tab first
+        createFromURLPage.clickCreateNewProjectTab();
+        projectHelpers.createProject(project, 'project/' + project['name'] + 'create/fromimage' + qs);
+        matchersHelpers.expectHeading(heading);
+        projectHelpers.deleteProject(project);
+      });
+
+      it('should load the image stream in to an existing project and verify the query string params are loaded in to the corresponding form fields', function(){
+        let createFromURLPage = new CreateFromURLPage();
+        projectHelpers.visitCreatePage();
+        projectHelpers.createProject(project);
+        createFromURLPage.visit(qs);
+        createFromURLPage.selectExistingProject(timestamp, uri);
+        matchersHelpers.expectHeading(heading);
+        let nameInput = element(by.model('name'));
+        expect(nameInput.getAttribute('value')).toEqual(name);
+        let sourceURIInput = element(by.model('buildConfig.sourceUrl'));
+        expect(sourceURIInput.getAttribute('value')).toEqual(sourceURI);
+        // click "Show advanced..." link to reveal hidden fields
+        element(by.css('[ng-click="advancedOptions = !advancedOptions"]')).click();
+        let sourceRefInput = element(by.model('buildConfig.gitRef'));
+        expect(sourceRefInput.getAttribute('value')).toEqual(sourceRef);
+        let contextDirInput = element(by.model('buildConfig.contextDir'));
+        expect(contextDirInput.getAttribute('value')).toEqual(contextDir);
+        projectHelpers.deleteProject(project);
+      });
+
+    });
+
+    describe('using a template supplied as a query string param', function() {
+
+      let sourceURL = "https://github.com/openshift/nodejs-ex.git-edited";
+      let qs = '?template=nodejs-mongodb-example&templateParamsMap=%7B"SOURCE_REPOSITORY_URL":"' + sourceURL + '"%7D' + '&namespace=' + namespaceForFixtures;
+      let uri = 'project/' + project.name + 'create/fromtemplate' + qs;
+      let heading = 'Node.js + MongoDB (Ephemeral)';
+      let words = project.name.split(' ');
+      let timestamp = words[words.length - 1];
+
+      it('should display details about the template', function() {
+        let createFromURLPage = new CreateFromURLPage();
+        createFromURLPage.visit(qs);
+        matchersHelpers.expectHeading(heading);
+      });
+
+      it('should load the template in to a newly created project', function() {
+        let createFromURLPage = new CreateFromURLPage();
+        createFromURLPage.visit(qs);
+        // NOTE: if the tabs are on the page because there are existing projects, click the 'Create New Project' tab first
```

Comment:
Same as above, i think we changed this to not use the `if/else` so can delete the comment.